static/usage/v7/fab/safe-area/angular/global_css.md

```diff
@@ -0,0 +1,16 @@
+```css
+:root {
+  /**
+   * Setting the variables for DEMO purposes only.
+   * Values will be set automatically when building an iOS or Android app.
+   */
+  --ion-safe-area-top: 20px;
+  --ion-safe-area-bottom: 20px;
+  --ion-safe-area-left: 20px;
+  --ion-safe-area-right: 20px;
+}
+
+ion-fab {
+  margin-top: var(--ion-safe-area-top, 0);
+}
```

Comment:
Should we separate this CSS out into the example's component file, that way it doesn't apply to _all_ `ion-fab` elements if they just had one page where a header was missing, for example.